src/app/js/formats/vega/component/tree-map/TreeMapIcon.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import ViewQuiltIcon from '@mui/icons-material/ViewQuilt';
+
+const TreeMapIcon = (props) => (
+    // We size the fontSize to empty string to disable it
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do not leave false praise (which can actually be damaging). Do look for something to sincerely praise.  -->
**praise:** ‏Thanks for the comment, otherwise it would only be a silent workaround